mbq/ranch/lib/error_handling.py

```diff
@@ -12,6 +15,13 @@ def log_errors_and_send_to_rollbar(func):
     def wrapper(*args, **kwargs):
         try:
             func(*args, **kwargs)
+        except psycopg2.InterfaceError:
```

Comment:
This won't work in os-core cuz it uses mysql.

Django should be wrapping the exception in a db-agnostic way. We should catch that instead.

Reply:
🤦‍♀️ Right. I was staring at the django wrapper exception in papertrail the whole time I was writing this PR, and still used psycopg instead